pkg/scheduler/scheduler.go

```diff
@@ -411,6 +411,13 @@ func (e entryOrdering) Less(i, j int) bool {
 }
 
 func (s *Scheduler) requeueAndUpdate(log logr.Logger, ctx context.Context, w *workload.Info, message string) {
+	if message == "cohort used in this cycle" {
```

Comment:
lets add a flag to `entry` and pass entry instead of workload.Info instead of this.

Reply:
Done